core-plugins/src/test/java/co/cask/hydrator/plugin/transform/LogParserTransformTest.java

```diff
@@ -47,6 +47,33 @@
   private static final Transform<StructuredRecord, StructuredRecord> CLOUDFRONT_TRANSFORM =
     new LogParserTransform(CLOUDFRONT_CONFIG);
 
+  private static final Schema LOG_SCHEMA = Schema.recordOf(
+    "event",
+    Schema.Field.of("uri", Schema.of(Schema.Type.STRING)),
+    Schema.Field.of("ip", Schema.of(Schema.Type.STRING)),
+    Schema.Field.of("browser", Schema.of(Schema.Type.STRING)),
+    Schema.Field.of("device", Schema.of(Schema.Type.STRING)),
+    Schema.Field.of("httpStatus", Schema.of(Schema.Type.INT)),
+    Schema.Field.of("ts", Schema.of(Schema.Type.LONG)));
+
+  @Test
+  public void testConfigurePipelineSchemaValidation() throws Exception {
+    MockTestConfigurer mockConfigurer = new MockTestConfigurer(Schema.of(Schema.Type.BYTES));
```

Comment:
we expect our schemas to be of type RECORD.  This is a good test case to have though, should check that this throws an exception.